### PR TITLE
add rnnlm recipe for librispeech

### PR DESCRIPTION
**Pull request by keli78**
will update rescoring results soon.

### CODE REVIEW

File: egs/librispeech/s5/local/rnnlm/tuning/run_tdnn_lstm_1a.sh [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+# Copyright 2012  Johns Hopkins University (author: Daniel Povey)
+#           2018  Ke Li
+
+# This script trains LMs on the librispeech 960 hours training data.
+
+# rnnlm/train_rnnlm.sh: best iteration (out of 26) was 21, linking it to final iteration.
+# rnnlm/train_rnnlm.sh: train/dev perplexity was 118.4 / 152.6.
+# Train objf: -5.74 -5.51 -5.38 -5.29 -5.22 -5.16 -5.12 -5.08 -5.05 -5.02 -4.99 -4.97 -4.97 -4.93 -4.90 -4.87 -4.84 -4.82 -4.79 -4.77 -4.75 -4.73 -4.71 -4.69 -4.67
+# Dev objf:   -6.00 -5.61 -5.45 -5.36 -5.29 -5.24 -5.20 -5.18 -5.16 -5.13 -5.12 -5.11 -5.11 -5.09 -5.07 -5.06 -5.05 -5.04 -5.03 -5.03 -5.03 -5.03 -5.03 -5.03 -5.03 -5.03
+
```
Comment (danpovey) [inline egs/librispeech/s5/local/rnnlm/tuning/run_tdnn_lstm_1a.sh]: Can you please include some WER comparisons?

Comment (keli78) [reply egs/librispeech/s5/local/rnnlm/tuning/run_tdnn_lstm_1a.sh]: sure, I have WER numbers on an old chain model. I'm running a new one now. will update the rescoring results on that in a few days.

**Comment by jtrmal** [comment]
I'm closing and opening just to force status change on travis

File: egs/librispeech/s5/local/rnnlm/tuning/run_tdnn_lstm_1a.sh [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+  for decode_set in test_clean test_other dev_clean dev_other; do
+    decode_dir=${ac_model_dir}/decode_${decode_set}_${LM}
+
+    # Lattice rescoring
+    rnnlm/lmrescore$pruned.sh \
+      --cmd "$decode_cmd --mem 8G" \
+      --weight 0.4 --max-ngram-order $ngram_order \
+      data/lang_test_$LM $dir \
+      data/${decode_set}_hires ${decode_dir} \
+      exp/decode_${decode_set}_${LM}_${decode_dir_suffix}_rescore
```
Comment (GaofengCheng) [inline egs/librispeech/s5/local/rnnlm/tuning/run_tdnn_lstm_1a.sh]: @keli78 put the rescore dir under exp? or for example under exp/chain_cleaned/tdnn_lstm_1n_sp/?

Comment (keli78) [reply egs/librispeech/s5/local/rnnlm/tuning/run_tdnn_lstm_1a.sh]: will modify this when I update the WERs. Thanks!

**Comment by keli78** [comment]
hi Dan @danpovey, I added the rescoring results and this PR is ready to be merged.

**Pull request merged**